HHS/OPRE-OPS
# feat: :sparkles: adds ServicesComponentAccordion

Refactor StepCreateBLIsAndSCs component to group budget lines by services component

## What changed

Adds ServicesComponentAccordion

## Issue

#1967 #2015 

## How to test

1. See Agreement one in Edit wizard
2. Skip Step 2
3. See ServicesComponentAccordions for 3 BLIs
4. add a BLI with no Services Component
5. See BLI with no Services Component in its own group

## Screenshots

<img width="1080" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/c2347d6f-5d47-4478-b7c5-451d2e397006">

## Definition of Done Checklist
